mlrun/api/utils/scheduler.py

```diff
@@ -258,12 +260,19 @@ def get_schedule(
         name: str,
         include_last_run: bool = False,
         include_credentials: bool = False,
+        transform_and_enrich: bool = False,
+        ensure_auth_info: bool = False,
+        auth_info: mlrun.api.schemas.AuthInfo = None,
```

Comment:
I think default should be True to be backwards compatible and  because we would like that to be performed always (atleast for now )

Reply:
not all callees have authinfo handy (see `pipelines.py` `_RemoteRunner.run` signature).

Reply:
I actually dont like this idea.
it mixes up getting schedule and enriching the auth info
i will revert and find a better way unifying the sync operations